v1.1/connection-parameters.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Client Connection Parameters
+summary: This page describes the parameters used to establish a client connection.
+toc: false
+---
+
+Client applications, including client [`cockroach`
+commands](cockroach-commands.html), work by establishing a network
+connection to a CockroachDB cluster. The client connection parameters
+determine which CockroachDB cluster they connect to, and how to
+establish this network connection.
+
+<div id="toc"></div>
+
+
+## Supported Connection Parameters
+
+There are two principal ways a client can connect to CockroachDB:
+
+- Most client apps, including most `cockroach` commands, use a SQL connection
+  established via a [PostgreSQL connection URL](#connect-using-a-url). When using a URL,
+  a client can also specify SSL/TLS settings and additional SQL-level parameters. This mode provides the most configuration flexibility.
+- Most `cockroach` commands also provide [discrete connection parameters](#connect-using-discrete-parameters) that
```

Comment:
Instead of `discrete connection parameters`, I'd prefer we be direct and say `command-line flags`? 

If you're on board, please change it throughout this page.